### DISCUSSION TITLE
No stereo instrument AU

### DISCUSSION THREAD

**Discussion by GMMPRM** [apohl79/audiogridder#960] [Help]
Hi all.
Just noticed that in Logic 10.7.4 I can only select a mono instrument instance (or multi) but not stereo. Is this expected behaviour? Running latest beta of plugin & server on Big Sur 11.6.5.

Cheers

**Comment by steveschow** [comment]
I'm getting the same issue.

**Comment by apohl79** [comment]
Hmmm, never had that issue. Will try a fresh install, when I get a chance.

**Comment by steveschow** [comment]
I will also comment that under DP11, it does show up as an available stereo VST...as well as stereo AU.  only Under LogicPro, it seems,...it shows up as mono.

**Comment by apohl79** [comment]
Issue confirmed (introduced in beta9 I believe) and fixed in the upcoming version.

**Reply by zen-archer** [reply]
Longtime happy user here who has just started to use the instrument in Logic 10.7.9 and it still only is available as mono or multi. Am I missing something? I am using the 1.2.0 latest stable version.

M1/macOS 13.4.1

**Comment by laurencebush** [comment]
Same problem in the latest version. Is there any way around it? It seems to be something to do with Apple, as just tried 1.1 and it has the same problem

**Comment by zen-archer** [comment]
FYI this now works with Logic Pro 11
